connectors/src/connectors/google_drive/temporal/activities.ts

```diff
@@ -224,32 +223,60 @@ async function syncOneFile(
   dataSourceConfig: DataSourceConfig,
   file: GoogleDriveFileType
 ) {
-  const docs = google.docs({ version: "v1", auth: oauth2client });
-  const res = await docs.documents.get({
-    documentId: file.id,
-  });
-  if (res.status !== 200) {
-    throw new Error(
-      `Error getting Google document. status_code: ${res.status}. status_text: ${res.statusText}`
-    );
-  }
-  if (!res.data.title) {
-    throw new Error("No title found");
+  const mimeTypesToExport: { [key: string]: string } = {
+    "application/vnd.google-apps.document": "text/plain",
+    "application/vnd.google-apps.spreadsheet": "text/csv",
+    "application/vnd.google-apps.presentation": "text/plain",
+  };
+  const mimeTypesToDownload = ["text/plain", "text/csv"];
+
+  let documentContent: string | undefined = undefined;
+  if (mimeTypesToExport[file.mimeType]) {
+    const drive = await getDriveClient(oauth2client);
```

Comment:
Just a question, no need to add a commit: could we directly pass `driveClient` instead of `oauth2client` to `syncOneFile`?